src/main/java/io/strimzi/kafka/bridge/amqp/AmqpConfig.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2016 Red Hat Inc.
```

Comment:
Do we have to update these copyright dates to 2019 or can we keep them at the original date issued?

Reply:
I was more thinking that we should start to use the same as in Strimzi, which is different from this one. I would live this for a different simple PR. Wdyt?

Reply:
Sounds good to me